### DISCUSSION TITLE
Freeplane beeps a puts error messages in the log. What can I do?

### DISCUSSION THREAD

**Discussion by peterstev** [freeplane/freeplane#1391] [Help]
Hi,
I am new to the community, so apologies if this wrong way to ask the question.

With increasing frequency, freeplane beeps and puts the following error messages into the log. These messages repeat a lot. What is the problem? What can I do?

Thank in advance
Peter

STDERR: java.lang.NoSuchMethodError: Lcom/apple/eawt/event/GestureHandler;.handleGestureFromNative(Ljava/awt/Window;IDDDD)V

**Comment by doctomlug** [comment]
You can disable the error beep and configure the log / reporting behavior within the preferences. 
If you installed 1.11.7 I recommend to get back to .6 (much less errors and stable)

**Reply by peterstev** [reply]
Thanks for the tip! I'm on 1.11.6.

Is there any way to make the error not happen? Or is this just background noise that is not worth worrying about?

**Reply by doctomlug** [reply]
I realized that some things are aborted due to the error (e.g. converting branches to nodes + link). As I use the software heavily and have to rely on it working I reverted back to 11.6 and will stay there until the next stable release. I recommend using the portable versions as easy way to have multiple versions available in parallel (and be able to update without the admins on company equipment)

The error piep is disabled in any case ... just to annoying

**Comment by peterstev** [comment]
Thanks @doctomlug ! I am using 11.6, though I generally follow the update recommendations. The advice to turn it off the sound in the settings is very helpful.

Is this issue on the radar screen of the developers? I wouldn't even know what to report.

**Reply by doctomlug** [reply]
Dimitry already solved it => 1.11.7_02
I just did some tests because he intended to change some html related code. This works as well now (at least I think it does). 
This guy is amazingly quick!

**Reply by peterstev** [reply]
Thank you, and thank you Dimitri. Looking forward to the next update!

**Comment by dpolivaev** [comment]
Hello folks,

1. I have not solved the issue mentioned in this question because it was never reported
2. If Freeplane makes the beeps it means that something works not as expected. The best you can do is submit a bug report and attach all log files.
3. I strongly recommend against deactivating the beep (and I even do not remember if it is possible) because the beep is an alarm and your work might be lost.
4. If you want me to analyze the problem please submit a proper issue clicking at https://github.com/freeplane/freeplane/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=  and attach the log files because they are essential for analysis
5. If you do not know where you can find the log files, post another question so that other users can help you with it.

Regards

**Reply by doctomlug** [reply]
Sorry, I commented not precisely enough. I meant to say that you solved the issue that I reported [#1381](https://github.com/freeplane/freeplane/issues/1381).
